source/developers-guide/rest-api/models/index.md

```diff
@@ -562,6 +562,30 @@ subgroup: REST API
 | group                    | string                  |                                                 |
 | sendMail              | boolean                  |                                                 |
 
+## Payment
+
+* **Model:** Shopware\Models\Customer\Payment
```

Comment:
this model and this table don't exist in Shopware. therefore I will decline your PR. 

I'm not quite sure, what your intention was behind that, but feel free to open another PR :blush: :+1:

Reply:
`
### {
    "id":"",
    "name":"",
    "description":"",
    "template":"",
    "hide":"",
    "additionalDescription":"",
    "debitPercent":"",
    "surcharge":"",
    "surchargeString":"",
    "position":"",
    "active":"",
    "esdActive":"",
    "mobileInactive":"",
    "pluginId":""
}
--

So this is the JSon structure I'm getting back from calling http://sw-demo.dev.1drop.de/api/orders/2 (i left out the values in the response).

I didn't find a model, that matches this structure but in the docs, Payment is mentioned and linked to (at least it tries). So I thought it's missing, isn't it :smile:?

`

Reply:
this looks like `\Shopware\Models\Payment\Payment` and the table `s_core_paymentmeans`
It's missing, you are right. 

indeed the link on this page https://developers.shopware.com/developers-guide/rest-api/api-resource-orders/ seems to be wrong. `Payment` redirects to `Payment-Instance` which is wrong.

Maybe you can change these both things? then we have the correct state :blush: :+1: